src/i18n/strings.ftl

```diff
@@ -1022,3 +1022,6 @@ JOURNAL-POSTS-WITHOUT-NUMBER = {$count ->
     [one] JOURNAL POST
     *[other] JOURNAL POSTS
 }
+
+Internet-Connection-Required = Internet Connection Required
```

Comment:
Please add a context comment here, e.g. "Title for dialog telling the user that an Internet connection is required"